.github/workflows/scripts/generate_release_notes.rb

```diff
@@ -55,13 +55,24 @@ def build_release_content
     FRONTMATTER
   end
 
+  def hyphenated_version_string
+    NewRelic::VERSION::STRING.tr('.', '-')
+  end
+
   def write_filename
-    "ruby-agent-#{NewRelic::VERSION::STRING.tr('.', '-')}.mdx"
+    "ruby-agent-#{hyphenated_version_string}.mdx"
   end
 
   def write_output_file
     File.write(write_filename, build_release_content)
   end
-end
 
-GenerateReleaseNotes.new.write_output_file
+  # The following 2 methods are used for dynamic naming in release_notes.yml
+  def file_name
+    p(write_filename)
```

Comment:
For consistency's sake, I think:

a) This line should be `puts write_filename` to use `puts` instead of `p`
b) The naming of "filename" vs "file_name" should be reconciled